.gitignore

```diff
@@ -9,3 +9,5 @@
 /.idea
 /.coverage*
 /htmlcov
+.cache/*
```

Comment:
I was running the tests locally and getting annoyed that this stuff kept getting making my repo dirty. Not directly related, but should have been there before.